src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -2385,12 +2385,12 @@ public void close() throws IOException {
             }
         }
 
-        private class CacheUdatingIterable extends SinglePassIterable<GHPullRequest> {
+        private class CacheUpdatingIterable extends SinglePassIterable<GHPullRequest> {
```

Comment:
AFAIK renaming this is not an issue as it's only used internally. Please double check.

Reply:
Guaranteed. It's a private class.  If someone is reflecting to get it, we cannot save them from themselves.

Reply:
Yeah, it's `private`, and as far as I can see, is not part of any serialized data, so we don't need to worry about compatibility from that perspective either.